src/components/ContributePrompt.js

```diff
@@ -23,7 +23,8 @@ export default class ContributePrompt extends Component {
     onLogin: PropTypes.func.isRequired,
     session: PropTypes.shape({
       isAnonymous: PropTypes.bool,
-      username: PropTypes.string
+      username: PropTypes.string,
+      publicEmails: PropTypes.bool
```

Comment:
Is it bool or an array?

Reply:
The service only expose if the user has at least one public email, it never shows which are those emails.